tests/ui/suggestions/invalid-bin-op.stderr

```diff
@@ -16,6 +16,10 @@ help: consider annotating `S<T>` with `#[derive(PartialEq)]`
 LL + #[derive(PartialEq)]
 LL | struct S<T>(T);
    |
+help: consider introducing a `where` clause, but there might be an alternative better way to express this requirement
+   |
+LL | pub fn foo<T>(s: S<T>, t: S<T>) where S<T>: PartialEq {
```

Comment:
Why is this being suggested now?

Reply:
It was removed in a const-traits PR: https://github.com/rust-lang/rust/commit/d464dd07ed83cc25ee3e568c98e3d4e26ff364ae#diff-a4f9511c26556adec6fac6f4073f446a12e9c9c883b0fd5ae672956caa5836c9 in #118661. This has been suggested before `PartialEq` gained the effect param, and adding the effect param in that commit made `PartialEq` not suggestable (which this PR makes it suggestable again)